pkg/sql/distsqlrun/base_test.go

```diff
@@ -0,0 +1,73 @@
+// Copyright 2017 The Cockroach Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+// implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package distsqlrun
+
+import (
+	"fmt"
+	"sync"
+	"testing"
+	"unsafe"
+
+	"github.com/cockroachdb/cockroach/pkg/sql/sem/tree"
+	"github.com/cockroachdb/cockroach/pkg/sql/sqlbase"
+)
+
+// Benchmark a pipeline of RowChannels.
+func BenchmarkRowChannelPipeline(b *testing.B) {
+	columnTypeInt := sqlbase.ColumnType{SemanticType: sqlbase.ColumnType_INT}
+
+	for _, n := range []int{1, 2, 3, 4} {
+		b.Run(fmt.Sprint(n), func(b *testing.B) {
```

Comment:
It'd add some immediate clarity if you renamed `n` to `length` and `fmt.Sprint(n)` to `fmt.Sprintf("length=%d", n)`.

Reply:
Done.